spec/factories/recipes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :recipe do
+    title { 'MyString' }
+    ingredients { 'MyText' }
+    instructions { 'MyText' }
+    references { '' }
```

Comment:
I might just not be familiar with this factory_bot API, but what does this line do?

Reply:
I was going to say the same thing! I don't think I've seen `references` before and I couldn't find it in the documentation after a quick search. Is it trying to connect the recipe with a user?

Reply:
It fills in the value for the `references` field. (I'm guessing this is historical based on a typo in an earlier creation of the recipe model?)

Reply:
The purpose is to connect the recipe with a user, but I think Mike is right here 😅